ko/tutorials/tour/traits.md

```diff
@@ -41,4 +41,4 @@ language: ko
     true
     true
 
-윤창석, 이한욱 옮김
\ No newline at end of file
+윤창석, 이한욱 옮김
```

Comment:
Ah... normally GitHub indicates that visually a specific way. I guess the Unicode confused it.  No worries, then.